packages/jsActions/mobile-resources-native/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Fixed
+
+-   react-native-device-info updated to 8.7.1 to fix getDeviceName function. 
```

Comment:
We can be more generic here and mention `We've updated react-native-device-info library`

Reply:
Done 👍